tf-modules/cumulus_distribution/bin/build-tf-module.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+# This is meant to be run from the tf-modules/cumulus_distribution directory as part of the bamboo/create-release.sh script
+(cd ../../packages/s3-credentials-endpoint && npm run package)
+
+mkdir -p dist
+
+cp ../../packages/s3-credentials-endpoint/dist/lambda.zip lambda.zip
```

Comment:
I am assuming that we need to include the S3 credentials Lambda here. I'm not 100% sure though.

Reply:
no.  cumulus_distribution api only calls <prefix>-DistributionApiEndpoints lambda.

Reply:
https://github.com/nasa/cumulus/blob/master/tf-modules/cumulus_distribution/api.tf#L42-L44
filename         = "${path.module}/../../packages/api/dist/distribution/lambda.zip"

Reply:
Also, is it necessary to add tf-modules/cumulus_distribution/lambda.zip to the baseline?

Reply:
also tf-modules/distribution/lambda.zip?

Reply:
Ah ok. I removed the S3 Credentials Lambda, and the bucket_map.yaml because I don't think that's going to be relevant now. The command here now just grabs all of the terraform files:

https://github.com/nasa/cumulus/pull/2279/commits/4bf77cd6471064218eb245fb85b1e51977d62ebc

Does that seem right? 

You asked about `tf-modules/distribution/lambda.zip`. I don't quite follow. Are you thinking we don't need to include the S3 credentials Lambda in `tf-modules/distribution` (TEA)?

Reply:
I meant you have this file in your PR, and I think we shouldn't add it to baseline.  If you go to [Files Changed](https://github.com/nasa/cumulus/pull/2279/files)
 `BIN +5.62 MB tf-modules/distribution/lambda.zip`

Reply:
I dtk how, you still need to include the zip file for creating prefix-DistributionApiEndpoints,
filename = "${path.module}/../../packages/api/dist/distribution/lambda.zip"

Reply:
Here's the change to include the distribution lambda.zip with cumulus_distribibution:

https://github.com/nasa/cumulus/pull/2279/commits/dbcda92e4fbff3ef9a9c70103429dbbc1d2bc778

That builds the whole /packages/api package and copies /packages/api/dist/distribution/lambda.zip to `dist/terraform-aws-cumulus-distribution.zip`. It's testable by running `./bamboo/create-release-artifacts.sh` and inspecting `cumulus_distribution/dist/terraform-aws-cumulus-distribution.zip`. In there is another lambda.zip with the correct lambda.